test/run-functional-tests

```diff
@@ -201,6 +201,34 @@ deploy_ha_node() {
 	info "Building crmsh on \"$node_name\"..."
 	docker_exec $node_name "$make_cmd" 1> /dev/null || \
 		fatal "Building failed on $node_name!"
+
+	# Create alice,bob and carol. This should be in done the Dockerfile, TODO!
```

Comment:
Is that fine to do below steps in `run-functional-tests` file?
I think it's easier to change than in dockerfile

BTW, could you please create a new function for below steps instead of all in depoly_ha_node function?

Reply:
I want to make that in the future we use fewer `root` and more custom users, so that `alice` and `bob` have the same right as the root. But one step at a time, I will move those lines outside the `depoly_ha_node` , but not into the feature file, because they are hardly a part of the test.